packages/common/src/stream.ts

```diff
@@ -115,6 +115,7 @@ export interface LogEntry {
   cid: CID
   type: CommitType
   timestamp?: number
+  expirationTime?: number
```

Comment:
Could you add some documentation what each of these params are. I don't think it's very obvious for an onlooker.